src/utils/advanced-filters-utils.js

```diff
@@ -0,0 +1,9 @@
+export function isBase64Encoded(str) {
+  // Base64 encoded strings can only contain characters from [A-Za-z0-9+/=]
+  const base64RegExp = /^[A-Za-z0-9+/=]+$/;
+  return base64RegExp.test(str);
+}
+
+export function isEmptyObject(obj) {
+  return Object.keys(obj).length === 0;
+}
```

Comment:
I think that both of those could be handled by existing functions from framework.

Reply:
First function cannot be handled by by existing function from framework.
The second one - yes - I decided to use lodash and it works.